gulpfile.js

```diff
@@ -4,51 +4,51 @@
 
 /* global __dirname, require, process */
 
-var gulp = require('gulp');
-var karma = require('karma');
-var eslint = require('gulp-eslint');
-var watch = require('gulp-watch');
-var spawn = require('child_process').spawn;
+const gulp = require('gulp');
+const karma = require('karma');
+const eslint = require('gulp-eslint');
+const watch = require('gulp-watch');
+const spawn = require('child_process').spawn;
 
-var lintPaths = [
+const lintPaths = [
     'media/js/**/*.js',
     '!media/js/libs/*.js',
     'tests/unit/spec/**/*.js',
     'gulpfile.js'
 ];
 
-gulp.task('serve:backend', function () {
-    var devServerPort = process.env.PORT || 8000;
+gulp.task('serve:backend', () => {
+    const devServerPort = process.env.PORT || 8000;
     process.env.PYTHONUNBUFFERED = 1;
     process.env.PYTHONDONTWRITEBITECODE = 1;
-    spawn('python', ['manage.py', 'runserver', '0.0.0.0:' + devServerPort], {
+    spawn('python', ['manage.py', 'runserver', `0.0.0.0:${devServerPort}`], {
         stdio: 'inherit'
     });
 });
 
-gulp.task('media:watch', function () {
+gulp.task('media:watch', () => {
     return gulp.src('./media/**/*')
         .pipe(watch('./media/**/*', {
             'verbose': true
         }))
         .pipe(gulp.dest('./static'));
 });
 
-gulp.task('js:test', function(done) {
+gulp.task('js:test', (done) => {
```

Comment:
Just mentioning that parens aren't necessary with a single argument. I haven't used ES6 enough to have a strong preference, but to this point I've been omitting the parens - probably just because I can. I'm good either way here.